### PR TITLE
I added my name and user profile link

### PR DESCRIPTION
**Pull request by CCccc-76**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,4 @@
 # git-demo
 to demonstrate how to use git and GitHub
+
+-[Jacqueline Chen](https://github.com/CCccc-76)
```
Comment (mariumtapal) [inline README.md:4]: Could you add a space between the `-` and your name? Looks great otherwise!

**Review by mariumtapal — CHANGES_REQUESTED** [review]
Could you add a space between the `-` and your name? Looks great otherwise!

**Comment by CCccc-76** [comment]
Just revised it!

File: README.md (hunk 1 above)
Comment (CCccc-76) [reply README.md]: Just revised it!

**Review by mariumtapal — APPROVED** [review]

**Pull request merged**